usaspending_api/broker/management/commands/load_fsrs.py

```diff
@@ -101,7 +103,7 @@ def gather_shared_award_data(self, data, award_type):
         skip_count = 0
         for row in data:
             counter += 1
-            if counter % 1000 == 0:
+            if counter % 10000 == 0:
```

Comment:
Nickpicky but I'd suggest making this a constant for readability.

Reply:
Agreed, will make the change.